src/sinks/influxdb/logs.rs

```diff
@@ -163,7 +163,12 @@ impl HttpSink for InfluxDBLogsSink {
         let mut event = event.into_log();
 
         // Measurement
-        let measurement = encode_namespace(Some(&self.namespace), '.', "vector");
+        let name = "vector";
+        let measurement = if self.namespace.is_empty() {
+            name.to_string()
+        } else {
+            format!("{}{}{}", self.namespace, '.', name)
```

Comment:
Nit: could we move the `.` into the format string? Or, even better, use the `encode_namespace` utility function.